pkg/reconciler/taskrun/taskrun.go

```diff
@@ -469,9 +469,7 @@ func extracPlatform(tr *tektonapi.TaskRun) (string, error) {
 
 func (r *ReconcileTaskRun) handleHostAllocation(ctx context.Context, tr *tektonapi.TaskRun, secretName string, targetPlatform string) (reconcile.Result, error) {
 	log := logr.FromContextOrDiscard(ctx)
-
-	log.Info("attempting to allocate host")
-
+	log.Info("attempting to allocate host for platform:" + targetPlatform)
```

Comment:
Please use logr key/values instead of embedding values into the string
```suggestion
	log.Info("attempting to allocate host", "platform", targetPlatform)
```
Also, are you sure we don't already embed the platform value into the logger somewhere? We probably should once we know what the requested platform is, so it gets added to all logged lines

Reply:
No, practically we have only request name and namespace:
`{"request": {"name":"mpc-stg-test-repo-on-push-xh46v-build-images-1","namespace":"mshaposh-tenant"}}`
I'll commit your suggestion.